### ISSUE TITLE
Conan Essentials - Environment Setup Fails

### ISSUE THREAD

**Issue by TomSmithGR** [conan-io/training#59]
Hi, 

I am on win10, using docker desktop v20.10.0-rc1.

When running the steps in Environment setup, on the conan essentials course, I am unable to build the images with the current Dockerfile configuration. When I run docker-compose up -d, it fails with:

"Step 1/11 : FROM docker.bintray.io/jfrog/artifactory-cpp-ce:7.6.2
ERROR: Service 'jfrog-artifactory-training' failed to build : missing or empty Content-Length header"

This seems to be an issue with the image on bintray.io, as running "docker pull docker.bintray.io/jfrog/artifactory-cpp-ce:7.6.2" results in 
"Error response from daemon: missing or empty Content-Length header".

Modifying the dockerfiles to use docker.bintray.io/jfrog/artifactory-cpp-ce:current seems to resolve the issue.

**Comment by memsharded** [comment]
Hi @TomSmithGR 

I have just tried in Windows with ``docker pull docker.bintray.io/jfrog/artifactory-cpp-ce:7.6.2``, and it seems that it is working ok.
It is possible that it was a temporary glitch in Bintray, would you mind trying again, please?

**Comment by TomSmithGR** [comment]
Hi @memsharded,

Thanks for the help. I have just repeated the call and get the same response. 

"D:\conantraining>docker pull docker.bintray.io/jfrog/artifactory-cpp-ce:7.6.2
Error response from daemon: missing or empty Content-Length header"

it seems specific to that version, I can pull 7.6.1 fine, and 7.6.3 fine, but 7.6.2 repeatedly fails.

**Comment by solvingj** [comment]
Please try:

`docker rm docker.bintray.io/jfrog/artifactory-cpp-ce:7.6.2`

and then try docker pull again.

**Comment by TomSmithGR** [comment]
I just tried that, the response was "Error: No such image: docker.bintray.io/jfrog/artifactory-cpp-ce:7.6.2". I am getting the same issue trying to pull the image.

**Comment by solvingj** [comment]
Please try switching between back and forth between windows/linux container mode.  

Also, is there any other windows computer in the same local area network that you can test with?  

Also, is it possible that you have an docker proxy server in your network which your docker client is configured to use? (for example, artifactory)?

**Comment by TomSmithGR** [comment]
Hi, I tried switching between container modes, and that didn't help. I have also upgraded to the latest build of docker desktop. I have no containers running, and nothing on my network that should be messing with the response from the server. Its still happening on that specific version, and not on later releases, which is odd.

I tested pulling the same release into an azure VM and that worked fine.

I am going to assume that this is an issue specific to this machine, and just carry on with a newer build of artifactory. Thanks for all the help.
